### PR TITLE
Update the statistics: add boxes data

### PR DESCRIPTION
**Pull request by danieltjason**

### CODE REVIEW

File: src/controllers/stats.controller.ts [hunk 1]
```diff
@@ -221,3 +223,18 @@ statsController.get(
     }
   },
 );
+
+statsController.get('/market/chart', async (req, res) => {
+  try {
+    const { period } = validateMarketChartsSchema.validateSync({
+      ...req.query,
```
Comment (crazyoptimist) [inline src/controllers/stats.controller.ts]: NB: no need to destructure req.query

File: src/migration/1629190095399-AddAverageBlockSizeAndTransactionPerBlockAndTransactionFeeInLast24HourAndMemPool.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddAverageBlockSizeAndTransactionPerBlockAndTransactionFeeInLast24HourAndMemPool1629190095399
+  implements MigrationInterface
+{
+  name =
```
Comment (crazyoptimist) [inline src/migration/1629190095399-AddAverageBlockSizeAndTransactionPerBlockAndTransactionFeeInLast24HourAndMemPool.ts:6]: NB: wondering why is this global variable needed?

File: src/index.ts [hunk 3]
```diff
@@ -44,8 +46,13 @@ createConnection({
         origin: frontendUrl,
         methods: ['GET', 'POST'],
       },
+      transports: ['websocket', 'polling'],
     });
+    const pubClient = new RedisClient({ url: process.env.REDIS_URL });
+    const subClient = pubClient.duplicate();
 
+    io.adapter(createAdapter({ pubClient, subClient }));
+    io.adapter();
```
Comment (crazyoptimist) [inline src/index.ts]: Wondering why to call adapter second time without parameters? Could you add explaining comment to the code for future developers?

**Review by crazyoptimist — APPROVED** [review]

File: src/migration/1629190095399-AddAverageBlockSizeAndTransactionPerBlockAndTransactionFeeInLast24HourAndMemPool.ts (hunk 2 above)
Comment (danieltjason) [reply src/migration/1629190095399-AddAverageBlockSizeAndTransactionPerBlockAndTransactionFeeInLast24HourAndMemPool.ts:6]: This is the name of migration.
This file was created by this command.
`yarn typeorm migration:generate -n`
https://stackoverflow.com/questions/65222981/typeorm-synchronize-in-production

File: src/index.ts (hunk 3 above)
Comment (danieltjason) [reply src/index.ts]: Thank you. It's redundant code

File: src/controllers/stats.controller.ts (hunk 1 above)
Comment (danieltjason) [reply src/controllers/stats.controller.ts]: I just fixed. 
https://github.com/pastelnetwork/pastel-explorer-backend/commit/121e184e7f1413ec290f48833f9943831c101f4e

**Review by Dicklesworthstone — APPROVED** [review]

**Review by crazyoptimist — APPROVED** [review]

**Pull request merged**